.github/actions/test_manager_and_agent/entrypoint.sh

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env bash
+set -e
+
+echo `systemctl status`
```

Comment:
Why `echo command` instead of just `command`?

Reply:
These commands were placed with debug purposes on the initial commits. I'm removing them completely.  Done on 
11104d4